tee-worker/identity/client-api/parachain-api/package.json

```diff
@@ -5,7 +5,7 @@
     "main": "dist/src/index.js",
     "module": "dist/src/index.js",
     "sideEffects": false,
-    "version": "0.9.19-07",
+    "version": "0.9.20-03-next.0",
```

Comment:
Hi @jonalvarezz what does this "next.0" mean?

Reply:
Hi @BillyWooo, good question – it signals that this is a preview version of upcoming releases as it contains non-released features (i.e., Email identity). 

See https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/identity/client-sdk/packages/enclave/README.md#versions